README.md

```diff
@@ -1,9 +1,346 @@
+
+```
+sudo rm appointmentscheduling-1.13.0.omod
+```
+
+In order to avoid conflicting tables in the database, please drop the tables below. by running the following sql statements.
+
+```
+set foreign_key_checks=0;
+drop table if exists appointmentscheduling_appointment;
+drop table if exists appointmentscheduling_appointment_block;
+drop table if exists appointmentscheduling_appointment_request;
+drop table if exists appointmentscheduling_appointment_status_history;
+drop table if exists appointmentscheduling_appointment_type;
+drop table if exists appointmentscheduling_block_type_map;
+drop table if exists appointmentscheduling_provider_schedule;
+drop table if exists appointmentscheduling_provider_type_map;
+drop table if exists appointmentscheduling_time_slot;
+set foreign_key_checks=1;
+```
+
+# **Clinical Appointments Scheduling**
+
+# **About**
+
+The appointment scheduling module aims at supporting 3.x users in creating and tracking patient appointments in order to do proper planning and follow up for the services that are being offered in a clinical set-up.
+
+It will also support some other actions; Edit, Cancel and Synchronization of appointments.
+
+# **Where to find it**
+
+The widget will be available on the left nav home page, and it will be named **Clinical Appointments**.
+
+# **Appointments module tables**
+
+Below is a diagram to show a schema of the appointments scheduling module.
+
+![alt_text](https://drive.google.com/uc?export=view&id=1SjbUBRPScmOzhB8YEruM1IX6tbSY5a55 "image_tooltip")
+
+# **_What does the appointments scheduling module entail?_**
+
+- Appointments summary dashboard.
+- Creating appointments.
+- Edit appointments.
+- Update appointments.
+- Synchronize appointments.
+- Cancel appointments.
+
+# **_Clinical appointments summary dashboard._**
+
+On this summary dashboard, users can view the daily appointments patient lists with the various outcomes of the appointments lists on different tabs, i.e. booked appointments, missed appointments, completed and canceled appointments.
+
+![alt_text](https://drive.google.com/uc?export=view&id=1HMM2k_hD9z2FaPh0kdlClVcHwsVsWrhH "image_tooltip")
+
+**_Booked appointments_**
+
+Shows a patient list of appointments and service(s) booked for the day.
+
+## **_Missed appointments_**
+
+Shows a patient line list of appointments with service(s) that were missed for the day.
+
+## **Completed appointments**
+
+Shows a patient line list of appointments and service(s) that were honored for the day.
+
+## **Canceled appointments**
+
+Shows a patient line list of patients who have had their appointments canceled.
+
+# **_Add/Create appointments._**
+
+By use of a form, users will be able to create appointments at the patient chart level by clicking on the Add New appointment button, search for a patient and open the appointments form. \
```

Comment:
Same comment as above. Separate out the documentation of frontend to a different repo/wiki and from here, you may just provide a link. This module is a pure backend module and users should know that different frontends are pluggable.